ci-jobs/scripts/build-webdriveragent.js

```diff
@@ -13,6 +13,17 @@ async function buildWebDriverAgent (xcodeVersion) {
   xcodeVersion = xcodeVersion || await xcode.getVersion();
   log.info(`Building bundle for Xcode version '${xcodeVersion}'`);
 
+  // Clear WebDriverAgent from derived data
+  const derivedDataPath = path.resolve(os.homedir(), 'Library', 'Developer',
+    'Xcode', 'DerivedData');
+  log.info(`Clearing contents of '${derivedDataPath}/WebDriverAgent-*'`);
+  for (const wdaPath of
+    await fs.glob('WebDriverAgent-*', {cwd: derivedDataPath, absolute: true})
+  ) {
+    log.info(`Deleting existing WDA: '${wdaPath}'`);
+    await fs.rimraf(wdaPath);
```

Comment:
we could optimise the cleanup by wrapping all these promises with  B.all

Reply:
Do you want that to be in this PR?

Reply:
it's a simple change, but it's up to you to decide